python/dgl/frame.py

```diff
@@ -492,6 +495,17 @@ def unpin_memory_(self):
             self._data_nd = None
             self.pinned_by_dgl = False
 
+    def record_stream(self, stream):
+        """Record stream that is using the storage.
+        Does nothing if the backend is not PyTorch.
+
+        Parameters
+        ----------
+        stream : torch.cuda.Stream.
+        """
+        if F.get_preferred_backend() == 'pytorch':
+            self.data.record_stream(stream)
```

Comment:
Should we throw an error if the backend is not pytorch?

Reply:
Updated.